{{cookiecutter.folder_name}}/Makefile

```diff
@@ -72,8 +78,14 @@ tar : main
 	cd ${tardir} && tar -czf ../${outname}.tar.gz . && cd ..
 	rm -rf ${tardir}
 
+authlist :
+	pip install mkauthlist
```

Comment:
I think we only want to upgrade the version of `mkauthlist` when it is specifically requested by the `upgrade` rule of the Makefile.